doc/man7/EVP_KDF_X963.pod

```diff
@@ -0,0 +1,136 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF_X963 - The X9.63-2001 EVP_KDF implementation
+
+=head1 DESCRIPTION
+
+The EVP_KDF_X963 algorithm implements the key derivation function (X963KDF).
+X963KDF is used by Cryptographic Message Syntax (CMS) for EC KeyAgreement, to
```

Comment:
If this is used in CMS - why don't I see any changes in the CMS code?

Reply:
The CMS entry point is in crypto/ec/ecdh_kdf.c